### PR TITLE
got: update to 0.71.

### PR DESCRIPTION
**Pull request by omar-polo**
The latest version on OpenBSD is 0.72 but in the -portable branch a version was skipped by mistake and so we have a 0.71 that's actually 0.72.
Next -portable version will be 0.73 in par again with the OpenBSD version.

The patch was also upstreamed so now got should cross-compile just fine.

edit: the changelog can be consulted here: http://gameoftrees.org/releases/CHANGES

- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/got/template [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
 revision=1
 wrksrc=got-portable-${version}
 build_style=gnu-configure
-hostmakedepends="automake byacc pkg-config"
+hostmakedepends="byacc pkg-config"
 makedepends="libmd-devel zlib-devel libuuid-devel libbsd-devel ncurses-devel openssl-devel"
 short_desc="VCS which prioritizes ease of use and simplicity over flexibility"
 maintainer="Omar Polo <op@omarpolo.com>"
 license="ISC"
 homepage="https://gameoftrees.org"
```
Comment (classabbyamp) [inline srcpkgs/got/template:12]:
```suggestion
homepage="https://gameoftrees.org"
changelog="http://gameoftrees.org/releases/CHANGES"
```

Comment (paper42) [reply srcpkgs/got/template:12]: please add that line, but use https if possible

Comment (omar-polo) [reply srcpkgs/got/template:12]: Added.  Thanks, I didn't know about this variable! :)

**Comment by omar-polo** [comment]
Thanks :)

**Pull request merged**